pkg/updater/resultstore/resultstore_test.go

```diff
@@ -93,6 +95,61 @@ func timeMustText(t time.Time) string {
 	return string(s)
 }
 
+func TestExtractGroupID(t *testing.T) {
+	cases := []struct {
+		name string
+		tg   *configpb.TestGroup
+		pr   *processedResult
+		want string
+	}{
+		{
+			name: "nil",
+		},
+		{
+			name: "primary grouping BUILD by override config value",
+			tg: &configpb.TestGroup{
+				DaysOfResults:                   7,
+				BuildOverrideConfigurationValue: "test-key-1",
+				PrimaryGrouping:                 configpb.TestGroup_PRIMARY_GROUPING_BUILD,
+			},
+			pr: &processedResult{
+				InvocationProto: &resultstore.Invocation{
+					Id: &resultstore.Invocation_Id{
+						InvocationId: "id-1",
+					},
+					Properties: []*resultstore.Property{
+						{
+							Key:   "test-key-1",
+							Value: "test-val-1",
+						},
+					},
+					Name: invocationName("id-1"),
+					Timing: &resultstore.Timing{
+						StartTime: &timestamppb.Timestamp{
+							Seconds: 1234,
+						},
+					},
+				},
+			},
+			want: "test-val-1",
+		},
+	}
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			got := extractGroupID(tc.tg, tc.pr)
+			if tc.name == "fallback grouping LABELS" {
+				h := fnv.New32a()
+				h.Write([]byte("test-label-1"))
+				tc.want = tc.want + strconv.Itoa(int(h.Sum32()))
+			}
+			strconv.Itoa(int((fnv.New32a().Sum32())))
```

Comment:
we are not using fallback grouping by labels - so no need for those

Reply:
good call - did not catch that